miniapp/build.gradle

```diff
@@ -100,6 +100,7 @@ dependencies {
         compileOnly project(':analytics')
         testImplementation project(':analytics')
     }
+    compileOnly project(':admob-latest')
```

Comment:
This shouldn't be added as a dependency to `miniapp`. Instead, `miniapp` should be added as a dependency to `admob-latest`.